src/executors/ParallelExecutor.js

```diff
@@ -1,7 +1,20 @@
 import ExecutorAbstract from '../ExecutorAbstract';
 
 class ParallelExecutor extends ExecutorAbstract {
+  constructor(thread, executorArray) {
+    super(thread);
 
+    this.executorArray = executorArray;
+  }
+
+  _execute(doneHandler, failHandler) {
```

Comment:
ParallelExecutor hasn't been implemented yet.  Will address this when implementing.